apps/client/src/app/components/home-overview/home-overview.component.ts

```diff
@@ -69,6 +70,10 @@ export class HomeOverviewComponent implements OnDestroy, OnInit {
       .subscribe((impersonationId) => {
         this.hasImpersonationId = !!impersonationId;
 
+        this.unit = this.hasImpersonationId
+          ? '%'
+          : this.user?.settings?.baseCurrency;
```

Comment:
I'm not sure what the impersonation is in this lingo, but in 2.20 with zen mode I get:
![image](https://github.com/ghostfolio/ghostfolio/assets/11974489/700b98c9-6f55-42fd-b6b9-33a9f6eb2dbc)


Which is not true - the value is in %.

Reply:
Thank you @slagiewka. I have created a bug report #2624.